assume/strategies/extended.py

```diff
@@ -18,15 +22,29 @@ def calculate_bids(
         Takes information from a unit that the unit operator manages and
         defines how it is dispatched to the market
 
-        Return: volume, price
+        Returns a list of bids that the unit operator will submit to the market
+        :param unit: unit to dispatch
+        :type unit: SupportsMinMax
+        :param market_config: market configuration
+        :type market_config: MarketConfig
+        :param product_tuples: list of products to dispatch
+        :type product_tuples: list[Product]
+        :param kwargs: additional arguments
+        :type kwargs: dict
+        :return: orderbook
+        :rtype: Orderbook
         """
         bids = []
         for product in product_tuples:
             start = product[0]
             end = product[1]
 
-            min_power, max_power = unit.calculate_min_max_power(start, end)
-            current_power = unit.outputs["energy"].at[start]
+            min_power, max_power = unit.calculate_min_max_power(
+                start, end
+            )  # max_power describes the maximum power output of the unit
```

Comment:
I prefer longer comments to be before or after the code, most of the time.
`max_power describes the maximum power which can be generated throughout the timeframe from start to end`
would probably be better.

But it would also be better to have this comment inside of `calculate_min_max_power` so that one can hove above the method and see its documentation if one is unsure what the return params are..?